packages/hint-meta-charset-utf-8/package.json

```diff
@@ -6,13 +6,10 @@
     ],
     "timeout": "1m"
   },
-  "dependencies": {
-    "cheerio": "^1.0.0-rc.2"
-  },
   "description": "hint for best practices related to the meta charset",
   "devDependencies": {
+    "@hint/parser-html": "^2.0.2",
```

Comment:
Should we just add this dependency to `utils-tests-helpers` and add the `html` parser directly to the configurations?

Reply:
In my other PR we added `html` directly to any configuration passed to `utils-tests-helpers`.

That said, I'll still need `parser-html` as a `devDependency` here in order to get the correct types when listening for the `parse::end::html` event. I just don't have to explicitly specify it in the tests.

Reply:
Got it.